src/style.css

```
@@ -1 +1,48 @@
-/* add styles here */
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-empty.png);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
```

```suggestion
.stars--1 .stars__star:nth-child(-n+1),
... ,
.stars--5 .stars__star:nth-child(-n+5) {
```
Consider grouping your selectors with commas